client/src/datafiles/layouts/published/PublishedDetailLayout.tsx

```diff
@@ -117,6 +139,75 @@ export const PublishedDetailLayout: React.FC = () => {
             projectId={projectId}
             entityUuid={data.tree.children[0].uuid}
           />
+          <br />
```

Comment:
Can this markup be componentized so that it's not duplicated between Other and experimental/simulation/etc?